src/ui/Table/Table.tsx

```diff
@@ -0,0 +1,97 @@
+import { styled } from '../../stitches.config';
```

Comment:
We're going to use [the Table that I created for DistributionsPage](https://github.com/coordinape/coordinape/blob/main/src/pages/DistributionsPage/Table.tsx) instead. @teeolendo is reusing it in #883. If you want to keep using this one for AdminPage, can you move it to that folder?

Reply:
I just copied it for now. Later, I will utilize your implementation and remove this.